src/key-value-pairs/index.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { getBaseProps } from '../internal/base-component';
+import { applyDisplayName } from '../internal/utils/apply-display-name';
+import useBaseComponent from '../internal/hooks/use-base-component';
+import { KeyValuePairsProps } from './interfaces';
+import InternalKeyValuePairs from './internal';
+
+export { KeyValuePairsProps };
+
+export default function KeyValuePairs({ columns = 1, items, ...rest }: KeyValuePairsProps) {
+  const { __internalRootRef } = useBaseComponent('KeyValuePairs');
```

Comment:
ColumnLayout reports property metrics for the `columns` property, so we should do the same here (https://github.com/cloudscape-design/components/blob/a8fdb97a6add00c0afd55292c1dcda04aae3ccd8/src/column-layout/index.tsx#L20).

Reply:
updated